copier/main.py

```diff
@@ -157,6 +157,12 @@ class Worker:
             When `True`, allow usage of unsafe templates.
 
             See [unsafe][]
+
+        skip_answered:
+            When `True`, skip questions that have already been answered.
```

Comment:
Does not seem to belong to this PR?

Reply:
Ah, right :sweat_smile: Yes, please, another PR would be great (and immediately merged).

Reply:
Done there: https://github.com/copier-org/copier/pull/1562 👍🏼 

I'll rebase this one onto it after merge